index.html

```diff
@@ -0,0 +1,217 @@
+
+      <Section class="about-myself">
+        <article class="about">
+        <h2 class="title-card">About Myself</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <a class="link" href="#">LET'S CONNECT</a>
+        <ul class="social-icons">
+           <li><a href=""><img src="images/social-media-icon/tw.png" alt="twitter"></a></li>
+           <li><a href=""><img src="images/social-media-icon/linkedin.png" alt="linkedin"></a></li>
+           <li><a href=""><img src="images/social-media-icon/medium.png" alt="medium"></a></li>
+           <li><a href=""><img src="images/social-media-icon/github.png" alt="github"></a></li>
+           <li><a href=""><img src="images/social-media-icon/v.png" alt="v"></a></li>
+        </ul>
+        <button class="btn"><a href="">Get my resume</a></button>
+        <div class="language">
+        <h3>Languages</h3>
+        <img src="images/browser-icons/chev-dow.svg" alt="">
+        <ul>
+        </div>
+            <li class="skill-2"><img src="images/browser-icons/javascript.svg" alt="">Javascript</li>
+                <li class="skill-2"><img src="images/browser-icons/html.svg" alt="">HTML</li>
+                    <li class="skill-2"><img src="images/browser-icons/css.svg" alt="">CSS</li>
+            
+        </ul>
+        <div class="framwork">
+        <h3>Framework</h3>
+            <img src="images/browser-icons/chev-right.svg" alt="" />
+        </div>
+        <div class="separator-line"></div>
+        <div class="framwork">
+        <h3>Skills</h3>
+        <img src="images/browser-icons/chev-right.svg" alt="" />
+          </button>
+        
+        </div> 
+        <div class="separator-line"></div>
+      </article>
+      </Section>
+
+      <!-- -------------- Footer --------------- -->
+      <section class="fouter">
+        <footer>
+        <div class="contact">
+        <h2 class="title-footer">Contact me</h2>
+        <p>
+            If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+           I’d love to help with it!
+        </p>
+        </div>
+        <form action="" class="contact-form">
+                <input type="text" name="name" placeholder="Your Name" required />
+                <input
+                  type="email"
+                  name="email"
+                  placeholder="Your Email"
+                  required
+                />
+                <textarea
+                  name="message"
+                  cols="30"
+                  rows="6"
+                  placeholder="Write your message here"
+                  required
+                ></textarea>
+                <button class="btn"><a href="">Get in touch</a></button>
+        </form>
+
+        </footer>    
```

Comment:
- [ ] Kindly get rid of this section as it will be a part of next milestone , it will help the Code reviewer to see the changes you will introduce to the relevant branch , but I really appreciate you courage 💪 

![Capture4](https://github.com/jerymey/Portfolio-setup/assets/80458199/19b530d9-1853-4a76-b7c6-b26ab3a44612)